pkg/sqlite/settings_storage.go

```diff
@@ -0,0 +1,39 @@
+package sqlite
+
+import (
+	"github.com/Duneus/go-kudos/pkg/gokudos"
+	"github.com/jinzhu/gorm"
+)
+
+var _ gokudos.SettingsStorage = &SettingsStorage{}
+
+type SettingsStorage struct {
+	storage *gorm.DB
+}
+
+func NewSettingsStorage(storage *gorm.DB) *SettingsStorage {
+	return &SettingsStorage{storage: storage}
+}
+
+func (s *SettingsStorage) SetScheduleSettings(scheduleSettings gokudos.Settings) error {
```

Comment:
```suggestion
func (st *SettingsStorage) SetScheduleSettings(scheduleSettings gokudos.Settings) error {
```

And then change `settings2` to `s`